lib/progress_bar/enumerable.rb

```diff
@@ -0,0 +1,6 @@
+require_relative '../progress_bar'
+
+# FIXME: should there be a better method?..
+[Enumerable, Array, Hash, Range].each do |mod|
```

Comment:
Isn't just putting it in `Enumerable` enough, since the others include it already?